docs-source/content/userguide/managing-fmw-domains/soa-suite/Logging-Exporter-Setup.md

```diff
@@ -0,0 +1,126 @@
+* "snakeyaml-1.23.jar" from [Maven Central](https://search.maven.org/artifact/org.yaml/snakeyaml/1.23/bundle).  
+
+## Copy JAR files to the Kubernetes Weblogic Domain Pod
+
+Copy the weblogic-logging-exporter-0.1.1.jar and snakeyaml-1.23.jar files to the domain home folder in the Admin server pod.
+
+```
+$ kubectl cp snakeyaml-1.23.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+ 
+$ kubectl cp weblogic-logging-exporter-0.1.1.jar soans/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/
+```
+## Add a startup class to your domain configuration
+
+In the Administration Console, navigate to "Environment" then "Startup and Shutdown classes" in the main menu.  
+Add a new Startup class. You may choose any descriptive name and the class name must be "weblogic.logging.exporter.Startup".  
+
+![Add startup class](images/Logging-Exporter_add-startup-class.png)  
+
+Target the startup class to each server that you want to export logs from.  
+
+![Target the startup class](images/Logging-Exporter_target-startup-class.png)  
+
+You can verify this by checking for the update in your config.xml which should be similar to this example:
+```
+<startup-class>
+  <name>weblogic-logging-exporter</name>
+  <target>AdminServer,soa_cluster</target>
+  <class-name>weblogic.logging.exporter.Startup</class-name>
+</startup-class>
+```  
+
+## Update Weblogic Server CLASS Path.
+
+Copy setDomainEnv.sh from the pod to local folder.  
+```
+$ kubectl cp soans-au01/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/bin/setDomainEnv.sh .
+```  
+
+Modify setDomainEnv.sh to update the Server Class path.  
+```
+CLASSPATH=/u01/oracle/user_projects/domains/soainfra/weblogic-logging-exporter-0.1.1.jar:/u01/oracle/user_projects/domains/soainfra/snakeyaml-1.23.jar:${CLASSPATH}
+export CLASSPATH
+```  
+
+Copy back the modified setDomainEnv.sh to the pod.  
+```
+$ kubectl cp setDomainEnv.sh soans-au01/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/bin/setDomainEnv.sh
+```
+
+## Create a configuration file for the WebLogic Logging Exporter  
+
+Create a file named WebLogicLoggingExporter.yaml. Specify the elasticseach server host and port number.
+```
+weblogicLoggingIndexName: wls
+publishHost: elasticsearch.default.svc.cluster.local
+publishPort: 9200
+domainUID: soainfra
+weblogicLoggingExporterEnabled:  true
+weblogicLoggingExporterSeverity:  Notice
+weblogicLoggingExporterBulkSize: 2
+weblogicLoggingExporterFilters:
+- FilterExpression: NOT(MSGID = 'BEA-000449')
+```  
+
+Copy file to the domain folder in the weblogic server pod.  
+```
+$ kubectl cp WebLogicLoggingExporter.yaml soans-au01/soainfra-adminserver:/u01/oracle/user_projects/domains/soainfra/config/
+```  
+
+## Restart Weblogic Servers  
+
+To restart the servers, edit the domain and change serverStartPolicy to NEVER for the weblogic servers to shutdown.  
+```
+$ kubectl edit domain -n soans-au01
```

Comment:
I would suggest doing this as a `kubectl patch` so that you don't have to walk the user through editing the document.  For instance:

`kubectl patch domain soainfra -n soans-au01 --type='json' -p='[{"op": "replace", "path": "/spec/serverStartPolicy", "value": "NEVER" }]'`